addons/website_event/views/website_event.xml

```diff
@@ -267,7 +268,10 @@
 <template id="event_details">
     <t t-call="website_event.layout">
         <div itemscope="itemscope" itemtype="http://schema.org/Event" class="container">
-            <h1 itemprop="name" class="text-center" t-field="event.name"></h1>
+            <div class="text-center mt16 mb16">
```

Comment:
Yes agree with @hmo-odoo . no need to use this class and class="h1". 
better to change position of label.